internal/app/server/server.go

```diff
@@ -36,6 +40,13 @@ func Run(bootstrapConfig *bootstrapv1.Bootstrap,
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
+	// Initialize request aggregation mapper component.
+	requestMapper := mapper.NewMapper(aggregationRulesConfig)
```

Comment:
nit: I wouldn't move this up here so we can just exit on `if mode != "serve"`

Reply:
i reverted the placement to where it was before to keep the scope of PR limited.